SequenceOfNumbersModified.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+public class SequenceOfNumbersModified {
+  public static void main(String[] args) {
+    if (args.length == 0) {
+      System.out.println("You entered nothing with running program");
+      Scanner in = new Scanner(System.in);
+      System.out.println("Enter your sequence now (use space between numbers):");
+      String sequence = in.nextLine();
+      String numStr[] = sequence.split(" ");
+
+      if (numStr.length == 1) {
+        System.out.println("You entered only one number. It is not a sequence");
+        return;
+      }
+      
+      for (int i = 0; i < args.length; i++) {

```

Comment:
Did you ever test the code? you can reach line 17 only if `args.length == 0` (see line 5) so you'll never go into the cycle